app/src/app.js

```diff
@@ -176,32 +177,71 @@ const Espace = () => {
-          <Header
-            onClickBurger={() => {
-              setMenuVisible(!menuVisible);
+  if (environment === "production") {
+    return (
+      <>
+        <div style={{ display: "flex" }}>
+          <Drawer open={menuVisible} onOpen={setMenuVisible} />
+          <Content>
+            <Header
+              onClickBurger={() => {
+                setMenuVisible(!menuVisible);
+              }}
+            />
+            <Switch>
+              <SentryRoute path="/account" component={Account} />
+              <SentryRoute path="/phase1" component={Phase1} />
+              <SentryRoute path="/phase2" component={Phase2} />
+              <SentryRoute path="/phase3" component={Phase3} />
+              <SentryRoute path="/les-programmes" component={Engagement} />
+              <SentryRoute path="/preferences" component={Preferences} />
+              <SentryRoute path="/mission" component={Missions} />
+              <SentryRoute path="/candidature" component={Candidature} />
+              <SentryRoute path="/desistement" component={Desistement} />
+              <SentryRoute path="/diagoriente" component={Diagoriente} />
+              {youngCanChangeSession(young) ? <SentryRoute path="/changer-de-sejour" component={changeSejour} /> : null}
+              {ENABLE_PM && <SentryRoute path="/ma-preparation-militaire" component={MilitaryPreparation} />}
+              <SentryRoute path="/" component={Home} />
+            </Switch>
+          </Content>
+          <ModalCGU
+            isOpen={modal?.isOpen}
+            title={modal?.title}
+            message={modal?.message}
+            confirmText={modal?.confirmText}
+            onConfirm={() => {
+              modal?.onConfirm();
+              setModal({ isOpen: false, onConfirm: null });
             }}
           />
-          <Switch>
-            <SentryRoute path="/account" component={Account} />
-            <SentryRoute path="/phase1" component={Phase1} />
-            <SentryRoute path="/phase2" component={Phase2} />
-            <SentryRoute path="/phase3" component={Phase3} />
-            <SentryRoute path="/les-programmes" component={Engagement} />
-            <SentryRoute path="/preferences" component={Preferences} />
-            <SentryRoute path="/mission" component={Missions} />
-            <SentryRoute path="/candidature" component={Candidature} />
-            <SentryRoute path="/desistement" component={Desistement} />
-            <SentryRoute path="/diagoriente" component={Diagoriente} />
-            {youngCanChangeSession(young) ? <SentryRoute path="/changer-de-sejour" component={changeSejour} /> : null}
-            {ENABLE_PM && <SentryRoute path="/ma-preparation-militaire" component={MilitaryPreparation} />}
-            <SentryRoute path="/" component={Home} />
-          </Switch>
-        </Content>
+        </div>
+      </>
+    );
+  }
+
+  return (
+    <div className="block md:flex">
+      <div className="fixed top-0 left-0 z-10">
```

Comment:
Je te propose une petite modification en lien avec le commentaire que je t'ai laissé sur le fichier `app/src/components/navbar/index.js`:

```suggestion
      <div className="fixed top-0 left-0 z-10 right-0 md:right-auto">
```